addons/attributes/functions/fnc_buttonSkills.sqf

```diff
@@ -0,0 +1,36 @@
+/*
+ * Author: mharis001
+ * Handles clicking the skills button.
+ *
+ * Arguments:
+ * 0: Button <CONTROL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [CONTROL] call zen_attributes_fnc_buttonSkills
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+private _entity = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
+private _unit   = if (_entity isEqualType grpNull) then {leader _entity} else {_entity};
```

Comment:
Maybe a select?

Reply:
This is mostly personal preference, but `ARRAY select BOOL` does require building the array first. I think the rest of this component uses `if else` for attributes that work with multiple entity types.